README.md

```diff
@@ -95,10 +97,37 @@ After your Docker container successfully builds and is running locally you can s
 3. Inside the `launch.json` file, set the `configuration->service->serviceAccountName` attribute to the service account email you just created.
 
 ## Deploying your own instance with terraform
-To run the pipeline with terraform, create your own `terraform.tfvars` file in the same directory as the other terraform files. For each variable declared in `prototype_variables.tf` that doesn't have a default, add your own for testing. Typically your own variables should be unique and can just be prefixed with your name or ldap. There are some that have specific requirements like project ids, code paths, and image paths.
-
-Currently the setup deploys both a data ingestion cloud funtion and a data ingestion cloud run instance. These are duplicates of each other. Once we get the cloud run one fully working we will delete the cloud fuction, but for now if you want to not have a duplicate while developing you can just comment out the setup for whichever one you don't want to use in `prototype.tf`
-
-Note that currently terraform doesn't diff the contents of the functions/cloud run service, so to get them to redeploy you can either
-1. Call `terraform destroy` every time before `terraform apply`, or
-2. Use [`terraform taint`](https://www.terraform.io/docs/commands/taint.html) to mark a resource as requiring redeploy
+Before deploying, make sure you have installed Terraform and a Docker client (e.g. Docker Desktop). See [One time setup](#one-time-setup) above.
+1. Create your own `terraform.tfvars` file in the same directory as the other terraform files. For each variable declared in `prototype_variables.tf` that doesn't have a default, add your own for testing. Typically your own variables should be unique and can just be prefixed with your name or ldap. There are some that have specific requirements like project ids, code paths, and image paths.
```

Comment:
I got this error for a lot of them as well:
"Resource name must use only lowercase letters, numbers and '-'. Must begin with a letter and cannot end with a '-'. Maximum length is 63 characters."
I think we should be updating the descriptions for each of them in prototype_variables.tf, though rather than elaborating here.

Also, it may be worth considering having some of the names be fixed with just a prefix as the variable? Instead of state_names_scheduler_name, county_names_scheduler_name, county_adjacency_scheduler_name etc, we could have scheduler_name_prefix where we could put our LDAP or other info.

These comments are not so much actionable in this PR but just topics of future conversation :)

Reply:
For things like the error you describe, they are already documented in the cloud and terraform docs, so I don't think we need to be too diligent about replicating that documentation. But I think the variables.tf file would be the right place for those notes.

The prefix could be convenient in the future for sure. We will have to see what the HET tracker setup looks like and if that will make sense. For right now I kind of like the flexibility and being able to find the names of all my resources in one place in the .tfvars file.